src/app/workflow-editor/object-inspector/io-inspector/workflow-io-inspector.component.ts

```diff
@@ -355,4 +365,12 @@ export class WorkflowIOInspectorComponent extends DirectiveBase implements OnIni
     isFileType() {
         return this.port.type.type === "File" || (this.port.type.type === "array" && this.port.type.items === "File");
     }
+
+    setDisabledState(isDisabled: boolean) {
+        const excludedControls = ["batchType"];
```

Comment:
We do not need excluded controls, as Boban suggested, select input when its in readonly state it cannot be expanded (you cannot see dropdown). We will change that everywhere.